saas_server_backup_ftp/models/res_config.py

```diff
@@ -51,36 +51,31 @@ def get_values(self):
         return res
 
     def test_sftp_connection(self):
-        server = self.env["ir.config_parameter"].sudo().get_param(
-            "saas_server.sftp_server", default=None)
-        username = self.env["ir.config_parameter"].sudo().get_param(
-            "saas_server.sftp_username", default=None)
-        password = self.env["ir.config_parameter"].sudo().get_param(
-            "saas_server.sftp_password", default=None)
-        sftp_rsa_key_path = self.env["ir.config_parameter"].sudo(
-        ).get_param('saas_server.sftp_rsa_key_path')
+        server = self.env["ir.config_parameter"].get_param("saas_server.sftp_server", default=None)
```

Comment:
Pliz keep 80 chars max for line :)

Reply:
But I kept 80 characters in the lines mentioned, incidentally, in the lines I mentioned I did not change anything.